examples/src/ray_traced_triangle/mod.rs

```diff
@@ -0,0 +1,459 @@
+use glam::{Mat4, Vec3};
+use std::time::Instant;
+use std::{env, mem};
+use wgpu::ray_tracing::{
+    AccelerationStructureFlags, AccelerationStructureUpdateMode, BlasBuildEntry, BlasGeometries,
+    BlasGeometrySizeDescriptors, BlasTriangleGeometry, BlasTriangleGeometrySizeDescriptor,
+    CommandEncoderRayTracing, CreateBlasDescriptor, CreateTlasDescriptor, DeviceRayTracing,
+    TlasInstance, TlasPackage,
+};
+use wgpu::util::{BufferInitDescriptor, DeviceExt};
+use wgpu::{include_wgsl, BufferUsages, SamplerDescriptor};
+
+struct Example {
+    tlas_package: TlasPackage,
+    compute_pipeline: wgpu::ComputePipeline,
+    blit_pipeline: wgpu::RenderPipeline,
+    bind_group: wgpu::BindGroup,
+    blit_bind_group: wgpu::BindGroup,
+    storage_texture: wgpu::Texture,
+    start: Instant,
+}
+
+#[repr(C)]
+#[derive(bytemuck::Pod, bytemuck::Zeroable, Clone, Copy, Debug)]
+struct Uniforms {
+    view_inverse: [[f32; 4]; 4],
+    proj_inverse: [[f32; 4]; 4],
```

Comment:
Implementing this gives a compile error saying that glam::Mat4 doesn't support Pod & Zeroable

Reply:
We just need to add the bytemuck feature to https://github.com/gfx-rs/wgpu/blob/trunk/examples/Cargo.toml#L37. This will enable their impl of Pod & Zeroable.

Reply:
This one still needs attention

Reply:
this should be fixed